### ISSUE TITLE
type annotation for createInstance() not prepared for lacking callback

### ISSUE THREAD

**Issue by jgehrcke** [googleapis/nodejs-bigtable#634]
This is analogous to https://github.com/googleapis/nodejs-bigtable/issues/564 which describes the same problem for `getInstances()`.

**Comment by laljikanjareeya** [comment]
Fixed by #650

**Comment by jgehrcke** [comment]
Not sure if this has really be resolved. Using the 3.0.1 release and I still have to do

```
//@ts-ignore
yield call([instance, "exists"]);
```

where `instance` is an object of type `Instance` from this package. The code works; without the `//@ts-ignore` we get

```
  The last overload gave the following error.
    Argument of type '(string | Instance)[]' is not assignable to parameter of type '{ context: unknown; fn: (this: unknown, ...args: any[]) => any; }'.

63       const result = yield call([instance, "exists"]);
                                   ~~~~~~~~~~~~~~~~~~~~

  ../../node_modules/@redux-saga/core/types/ts3.6/effects.d.ts:499:17
    499 export function call<Ctx, Fn extends (this: Ctx, ...args: any[]) => any>(
                        ~~~~
    The last overload is declared here.


Found 1 error.
```

Now, in `yield call([instance, "exists"]);` the `call()` is from redux-saga, and `call([instance, "exists"])` is precisely the recommended method to call a method on a class, also see https://github.com/redux-saga/redux-saga/issues/27.

Not sure who or what is to blame here, so please bear with me :).
